pkg/util/log.go

```diff
@@ -0,0 +1,71 @@
+package util
+
+import (
+	"fmt"
+	"io"
+	"net"
+
+	"github.com/google/uuid"
+	"github.com/sirupsen/logrus"
+)
+
+type Hook struct {
+	formatter logrus.Formatter
+	levels    []logrus.Level
+	fields    logrus.Fields
+	writer    io.Writer
+}
+
+var (
+	fieldMap = logrus.FieldMap{
+		logrus.FieldKeyTime:  "@time",
+		logrus.FieldKeyLevel: "@level",
+		logrus.FieldKeyMsg:   "message",
+	}
+	fields = logrus.Fields{
+		"@key": uuid.New(), // This will be unique for every command being run
```

Comment:
why is this here? Just curious. I'm out of touch of this feature and don't understand why you want this.

Reply:
It's a convenience to make it easier to follow what happens at each executed command rather than trying to work it out ourselves, because the logs easily get pretty garbled together otherwise. It's not on request, but this is a pretty standard feature when dealing with event based systems to make it easier to follow.